entity/src/product_status.rs

```diff
@@ -8,7 +8,7 @@ pub struct Model {
     pub advisory_id: Uuid,
     pub vulnerability_id: String,
     pub status_id: Uuid,
-    pub base_purl_id: Option<Uuid>,
+    pub component: Option<String>,
```

Comment:
I like the name much better

Reply:
I prefer component as packages are overloaded - though good as long as they are the same wherever (package or component).

Reply:
I changed them to package. I like the term component better as well, but let's discuss this on a whole code base level once we are ready to change that